### DISCUSSION TITLE
Isolate Container Tabs

### DISCUSSION THREAD

**Discussion by dannymate** [pulse-browser/browser#269] [General]
Vivaldi has [workspaces](https://vivaldi.com/features/workspaces/) for collections of Tabs. Like Containers these have a Name & Icon. I want to be able to select a Container type so only that containers Tabs are visible.

**Comment by sterno900** [comment]
this is what you are looking for https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups/
